css/templates/e-commerce/page.css

```diff
@@ -448,3 +467,21 @@ amp-selector [option][selected] {
     margin-right: calc(1/12 * 100%);
   }
 }
+
+
+/* PRODUCT LISTING */
+.commerce-listing-banner {
+  width: 100%;
+}
+
+@media (--breakpoint-md) {
+  .commerce-listing-header {
+    margin-top: var(--space-3);
+    border-top: 1px solid var(--color-separator);
+    border-bottom: 1px solid var(--color-separator);
+  }
+
+  .commerce-listing-wrapper {
+    margin-left: calc(1/12 * 100%);
```

Comment:
Had to find a way to use up the space of 1 column. Gutters don't seem to do what's needed. I've defined this declaration twice, should I make a --col-1 css var?